features/site.feature

```diff
@@ -130,7 +130,7 @@ Scenario: Editing a site's transition date as a non-GDS Editor
   And a site dclg exists
   And I visit this site page
   Then I should not see "Edit date"
-  When I visit the path /sites/dclg/edit
+  When I visit the path /site_dates/dclg/edit
```

Comment:
I wonder if the path would be better as `/sites/dclg/edit_date` as that would retain the path structure associating this with a site?

It'd also mean we wouldn't need a new controller, but could instead add `edit_date` and `update_date` methods to the `SitesController`.

Reply:
So I've always done this and added extra actions to existing controllers. However, I read this article (and I've seen similar others) which contains good arguments for not doing that http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/

Let me know what you think!

I do however think I should update the path at least!

Reply:
I'm sure we can have `/sites/dclg/edit_date` or `/sites/dclg/date/edit` despite it being a different controller (if you agree that that article makes sense)

Reply:
I can see a separate controller making make sense here.  I think `/sites/dclg/edit_date` would be my preferred URL, as there would be nothing rendered at `/sites/dclg/date`.

Reply:
Done as a new commit 💯

Reply:
Change looks good :+1:.  I wonder if the commit could be rebased into [`Add new controller for editing a sites transition date`](https://github.com/alphagov/transition/pull/1404/commits/0480316ad9fae66405e61c93340c278d30255ec8) so the commit history is more succinct?

Reply:
Done! :hurtrealbad: